samples/ai-gw-llm-proxy/run.sh

```
@@ -0,0 +1,94 @@
+
+print_title() {
+  echo
+  tput bold
+  tput setaf 2
+  echo "=== $1 ==="
+  tput sgr0
+  echo
+}
+
+if [ -f .env ]; then
+  . .env
```

Shellcheck (0.11.0)

[info] Not following: .env was not specified as input (see shellcheck -x).

(SC1091)